docs/intro.rst

```diff
@@ -486,10 +486,59 @@ HMC, or you can specify the HMC related parameters in the example scripts
 directly.
 
 
+.. _`HMC/SE version requirements`:
+
+HMC/SE version requirements
+---------------------------
+
+This section describes how dependencies of zhmcclient functionality to HMC or SE
+versions are documented.
+
+The following version related indicators are relevant for the HMC Web Services
+API:
+
+* **HMC version** (e.g. 2.16) - The version of the HMC. This corresponds to the
+  machine generation of the CPC with which the HMC was delivered and to the
+  "driver level" of the HMC, e.g. an HMC version 2.16 corresponds to z16 and
+  driver level D51C.
+
+* **HMC API version** (e.g. 4.10) - The HMC Web Services API has its own
+  version, independent of the HMC version.
+
+* **SE version** (e.g. 2.16) - The version of the SE. This corresponds to the
+  machine generation of the CPC and to the "driver level" of the SE/CPC, e.g.
+  an SE version 2.16 corresponds to z16 and driver level D51C.
+
+* **Firmware features** - This is the first feature concept that was introduced
+  with HMC version 2.14.0 and  HMC API version 2.23.
```

Comment:
Maybe add that this concept is deprecated.

Reply:
The HMC API book does not say that it is deprecated. And in 2.16, one more firmware feature was added. I understand the direction to use API features, but that is just not officially stated in the book.